README.md

```diff
@@ -29,69 +34,75 @@ See also `:help lspconfig`.
    ```
    nvim main.py
    ```
-4. Run `:LspInfo` to see the status or to troubleshoot.
```

Comment:
Huh, is there anything wrong with `:LspInfo`? Or is the information duplicated in `:checkhealth lsp`?

Reply:
Nothing is wrong with it, but this is step 1 in removing special snowflake functionality from nvim-lspconfig in favor of Neovim core alternatives.